service/src/main/java/org/apache/celeborn/server/common/service/config/SystemConfig.java

```diff
@@ -31,16 +31,19 @@ public class SystemConfig extends DynamicConfig {
   public SystemConfig(CelebornConf celebornConf, Map<String, String> configs) {
     this.celebornConf = celebornConf;
     this.configs.putAll(configs);
+    this.quota = currentQuota();
```

Comment:
Better use lazy initialization for quota

Reply:
> Better use lazy initialization for quota

Done